### DISCUSSION TITLE
https://github.com/settings/pages styling issue when you have no verified domains.

### DISCUSSION THREAD

**Discussion by 0neMiss** [community/community#76791] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

There is a newline after every character for me when opening github pages with no verified domains.

![image](https://github.com/community/community/assets/56180519/c1f5ea23-8b96-4a7b-b950-89ae00822b99)

reproduced on chrome Version 119.0.6045.159 (Official Build) (arm64)

Also reproduced on Safari Version 17.1 (19616.2.9.11.7)

and firefox 120.0 (64-bit)

The issue appears to be that the blankslate-container has no width.

![image](https://github.com/community/community/assets/56180519/883f71e2-08b6-4307-8915-a94eae2d8b3a)

**Comment by AlBovo** [comment]
I tried with the same Browsers you listed and they work just fine for me. Maybe they've patched the bug.

**Reply by 0neMiss** [reply]
I cant reproduce anymore either. closing this out
